### PR TITLE
修复 clash-meta 的 clash-meta@.service 文件错误

### PR DESCRIPTION
**Pull request by Se7entny**
clash-meta@sevice服务的CapabilityBoundingSet和AmbientCapabilities有问题，导致无法systemctl无法使用非root用户成功启动服务，这应修改为与Clash.Meta文档一样 https://github.com/MetaCubeX/Clash.Meta#general-installation-guide-for-linux

CapabilityBoundingSet=CAP_NET_ADMIN CAP_NET_RAW CAP_NET_BIND_SERVICE
AmbientCapabilities=CAP_NET_ADMIN CAP_NET_RAW CAP_NET_BIND_SERVICE

### CODE REVIEW

**Comment by Se7entny** [comment]
@sukanka

**Review by sukanka — APPROVED** [review]
Approved. 
I'm the maintainer of this pkg in both archlinuxcn and aur. But I have no permission to merge this, if a member sees this, please merge it. Thanks!

Reference: 
* https://github.com/archlinuxcn/repo/blob/master/archlinuxcn/clash-meta/lilac.yaml
* and https://aur.archlinux.org/packages/clash-meta

**Pull request merged**
